app/src/androidTest/java/sw19/moring03/paint/StrokeWidthTest.java

```diff
@@ -37,7 +37,7 @@ public void testStrokeWidthChange() {
         onView(withId(R.id.strokeWidthChooserButton)).perform(click());
         onView(withId(R.id.strokeWidth)).perform(changeValueOfStrokeWidthSeekBar(expectedStrokeWidth));
 
-        assertEquals(expectedStrokeWidth, launchActivityRule.getActivity().getStrokeWidth());
+        assertEquals(expectedStrokeWidth + 1, launchActivityRule.getActivity().getStrokeWidth());
```

Comment:
StrokeWidthChooserMenuBottomSheetDialog.java onProgressChanged add if stmt and change 0 to 1 if necessary